snapshotstore/sql/migrate.go

```diff
@@ -0,0 +1,28 @@
+package sql
+
+// Migrate the database
+func (sql *SQL) Migrate() error {
+	sqlStmt := `
+	create table snapshots (id VARCHAR NOT NULL, type VARCHAR, data BLOB);
+	create unique index id_type on snapshots (id, type);
+	delete from snapshots;
+	`
+	_, err := sql.db.Exec(sqlStmt)
+	return err
+}
+
+// MigrateTest remove the index that the test sql driver does not support
+func (sql *SQL) MigrateTest() error {
```

Comment:
This test merely test the SQL statements listed in the string slice,
thus it doesn't really test anything and certainly not the migration above.
Maybe remove it?

Reply:
This is not a test, but used in the test. Due to ramsql not supporting indexes.